source/boards/working-with-boards.rst

```diff
@@ -52,4 +74,48 @@ Table views
 
 Each column corresponds to a card property. You can edit cells directly or you can select **Open** to open the card editor for that row.
 
-Select the headers to sort them or insert new properties.
+=======
+
+All changes you make to boards and cards are saved immediately.
```

Comment:
I think this is ok. Note that changes to text fields (e.g. titles or card descriptions) are saved when exiting the control (when the control loses focus). Probably don't need that level of detail, but something to be aware of.

Reply:
That's a good point actually. I think it's worth adding actually, thank you!